src/analysis/normed_space/basic.lean

```diff
@@ -470,3 +477,21 @@ have h₂ : tendsto (λs:finset ι, s.sum (λi, ∥f i∥)) at_top (nhds (∑ i,
 le_of_tendsto_of_tendsto at_top_ne_bot h₁ h₂ $ univ_mem_sets' $ assume s, norm_triangle_sum _ _
 
 end has_sum
+
+namespace complex
+
+instance : normed_field ℂ :=
+{ norm := complex.abs,
+  dist_eq := λ _ _, rfl,
+  norm_mul := complex.abs_mul,
+  .. complex.discrete_field }
+
+lemma norm_real (r : ℝ) : ∥(r : ℂ)∥ = ∥r∥ := complex.abs_of_real _
+
+lemma norm_nat (n : ℕ) : ∥(n : ℂ)∥ = n := complex.abs_of_nat _
+
+lemma norm_int {n : ℤ} (hn : n ≥ 0) : ∥(n : ℂ)∥ = n :=
+suffices ∥((n : ℝ) : ℂ)∥ = n, by simpa,
+by rw [complex.norm_real, real.norm_eq_abs, _root_.abs_of_nonneg]; exact int.cast_nonneg.2 hn
+
```

Comment:
Does it make sense to also add `norm_rat`?
Should these be `simp`-lemmas?

Reply:
Yes, both makes sense.

Also I think I will change this by removing the assumption and replace the right side by `abs n`.